cmd/flow-aggregator/options.go

```diff
@@ -31,7 +31,8 @@ import (
 const (
 	defaultExternalFlowCollectorTransport = "tcp"
 	defaultExternalFlowCollectorPort      = "4739"
-	defaultFlowExportInterval             = 60 * time.Second
+	defaultActiveFlowRecordTimeout        = 60 * time.Second
+	defaultInactiveFlowRecordTimeout      = 120 * time.Second
```

Comment:
is there an RFC somewhere with a recommended values or did you come up with values you thought were appropriate?

Reply:
RFC does not suggest any values. Based on the values at Flow Exporter in Antrea agent, I came up with something appropriate.